test/phpunit/AllTests.php

```diff
@@ -26,6 +27,9 @@
 print "PHP Version: ".phpversion()."\n";
 print "Memory limit: ". ini_get('memory_limit')."\n";
 
+// Workaround for false security issue with main.inc.php in tests:
+$_SERVER['PHP_SELF'] = "phpunit";
```

Comment:
We should not force this value. This break test that must fails when value is wrong.
I see no reason to force his value to a dummy value. Dummy value may result in dummy test too.

Reply:
You can unforce it later once the security test issue is fixed - see #28214 .
This helps unblock that testing is failing on windows-ci.

Reply:
Regarding the regex: it's likely that but I did not examine the exact line of code.
The regex could be adapted to allow the case where we have a windows path : `/((?<!^[A-Za-z]):|[;"\'<>\?\(\){}\$%]/`